src/js/keyboard/keyboardHandler.js

```diff
@@ -0,0 +1,35 @@
+export class KeyboardHandler {
```

Comment:
This should just be merged in with `Shortcuts` and then renamed to `KeyboardShortcuts`